### DISCUSSION TITLE
old 16 bit exe. Patches not saved/applied

### DISCUSSION THREAD

**Discussion by das873** [NationalSecurityAgency/ghidra#6738] [Debugger]
Hi everyone,
I am a newbie in the degubbing/decompling and (especially) of Ghidra
I am trying to debug an ond 16-bit exe.
Once I modify some code I am not sure to be able to save the modified exe...
I have tried with the savepatch.py script but it save te unmodified version of the exe...

What are my errors in the procedure?
(Thank you in advance!)

**Comment by ryanmkurtz** [comment]
You want to do `File -> Export Program`.  For "Format", select `Original File`.  Note that this only will save the changes you made to bytes that are backed by the file.  If you, for example, add a new memory block...those new bytes won't be in your saved file.

**Comment by das873** [comment]
Thank you!!!
